### PR TITLE
update readme with info on working groups

### PR DESCRIPTION
**Pull request by anvega**
This pull request updates the README to include information about the Security TAG's working groups and community meetings. It provides details on how the working groups organize community activity, their focus areas, and the responsibilities of the working group leads. Additionally, it clarifies the types of deliverables produced by the working groups and where to find them in the repository. These changes aim to improve the clarity and completeness of the README for new and existing members of the Security TAG community.

 This update also creates a directory structure for each working group and their project files, improving organization and accessibility to these different resources.

### CODE REVIEW

File: community/supply-chain-security/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Software Supply Chain Security
+
+Software Supply Chain attacks have come to the wider community's attention following a recent high-profile attack, but have been an ongoing threat for a long time. With the ever-growing importance of free and open source software, software supply chain security is crucial, particularly in cloud native environments where everything is software-defined.
+
+## Meeting Information
+- **Weekly Meetings:** 8:00 AMP Pacific Time (US and Canada)
```
Comment (mnm678) [inline community/supply-chain-security/README.md:6]:
```suggestion
- **Weekly Meetings:** 8:00 AM Pacific Time (US and Canada) 
```

**Review by mnm678 — COMMENTED** [review]
Thanks for updating this! I'll give other WG leads a chance to review the new descriptions (cc @SantiagoTorres @JonZeolla @JustinCappos)

**Comment by anvega** [comment]
Certainly. The descriptions are not new; I extracted them either from the working group issue or from the existing content in the ongoing projects section.

File: README.md [hunk 2]
```diff
@@ -151,48 +152,22 @@ seen [here](governance/related-groups/README.md)
...
 
-See [CNCF calendar](https://calendar.google.com/calendar/u/0/embed?src=0b8u5el8ta4s93t2cm72tuvhhk@group.calendar.google.com&ctz=America/Los_Angeles) for invite.
+| Project                         | Leads                                       |
+|---------------------------------|---------------------------------------------|
+| [Applied Research](/community/research/README.md)               | Andrés Vega                                 |
+| [Automated Governance](/community/automated-governance/README.md)             | Andres Vega, Brand Keller                   |
```
Comment (JonZeolla) [inline README.md:164]:
```suggestion
| [Automated Governance](/community/automated-governance/README.md)             | Andrés Vega, Brandt Keller                   |
```

File: community/controls/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Controls
+
+## Overview
+
+This controls catalogue is focused on mapping to existing frameworks and regulations (CSA, NIST, FedRamp, SOX, GDPR, etc.), conversion to machine-readable format (OSCAL, JSON, etc.), inclusion of tests to validate/verify (both process and technical tests as appropriate), and application to security reviews to improve consistency of CNCF Security TAG reviews. It should also address requirements for Auditors for Cloud Native Platforms.
```
Comment (JonZeolla) [inline community/controls/README.md:5]:
```suggestion
The controls catalog ensures the usability of CNCF deliverables by GRC professionals and Auditors of Cloud Native systems. The primary goal is to support quickly identifying the controls status of cloud-native environments, and methods to share the output of those assessments in machine readable formats (OSCAL, JSON, etc.). These outputs should also map to existing frameworks and regulations (CSA, NIST, FedRAMP, SOX, GDPR, etc.), and provide guidance to properly validate and verify administrative and technical controls.
```

File: community/controls/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## Impact
+
+This initiative aims to help various stakeholders by providing a standardized set of controls mapped to relevant frameworks and regulations, enabling easier compliance and security assessments in cloud native environments.
```
Comment (JonZeolla) [inline community/controls/README.md:9]:
```suggestion
This initiative aims to provide a method for assessing environments against a standardized set of controls mapped to relevant frameworks and regulations, enabling easier compliance and security assessments in cloud native environments.
```

**Comment by anvega** [comment]
I made a mess attempting to squash commits having had the merges in between. I'll close this PR and reopen a clean one.

**Pull request closed without merging**